drivers/SmartThings/matter-switch/profiles/light-color-level-nl.yml

```diff
@@ -0,0 +1,25 @@
+name: light-color-level-nl
+components:
+- id: main
+  capabilities:
+  - id: switch
+    version: 1
+  - id: switchLevel
+    version: 1
+  - id: colorTemperature
+    version: 1
+  - id: colorControl
+    version: 1
+  - id: motionSensor
+    version: 1
+  - id: illuminanceMeasurement
+    version: 1
+  - id: firmwareUpdate
+    version: 1
+  - id: refresh
+    version: 1
+  categories:
+  - name: Light
+metadata:
```

Comment:
Remove the metadata from this profile so that the values will display correctly in the app.